src/patternfly/demos/MasterDetail/examples/MasterDetail.md

```diff
@@ -0,0 +1,404 @@
+          {{/drawer-content}}
+
+          <!-- Panel -->
+          {{#> drawer-panel
+            drawer-panel--type="div"
+            drawer-panel--attribute=(concat 'id="' master-detail-template--id '-panel" aria-label="Panel"')
+            drawer-panel--modifier="pf-m-width-75-on-xl"
+            drawer-panel--NoBody="true"
+            }}
+
+            <!-- Panel  -->
+            {{#> drawer-panel-body}}
+              {{> master-detail-content-header
+                master-detail-content-header--title="Patternfly-elements"
+                master-detail-content-header--sub-title="PatternFly elements"
+                }}
+            {{/drawer-panel-body}}
+            {{#> drawer-panel-body}}
+              {{> master-detail-content-body}}
+            {{/drawer-panel-body}}
+          {{/drawer-panel}}
+        {{/drawer-main}}
+      {{/drawer}}
+    {{/card}}
+  {{/page-main-section}}
+{{/master-detail-template}}
+```
+
+```hbs title=Master-detail-card-data-list isFullscreen
+{{#> master-detail-template
+  master-detail-template--id="master-detail-card-data-list-example"
+  master-detail-template--title="Master detail and simple list, in card"
+  master-detail-template--description="Body text should be Red Hat Text Regular at 16px. It should have leading of 24px because of its relative line height of 1.5"
+  }}
+
+  {{#> page-main-section}}
+    {{#> card}}
+
+      <!-- Drawer -->
+      {{#> drawer
+        drawer--id=(concat master-detail-template--id '-drawer')
+        drawer--IsStatic="true"
+        }}
+
+        {{#> drawer-main}}
+          <!-- Content -->
+          {{#> drawer-content drawer-content--NoBody="true"}}
+            {{#> drawer-content-body drawer-content-body--modifier="pf-m-no-padding"}}
+              {{> master-detail-card-toolbar}}
+              {{> master-detail-card-data-list}}
+            {{/drawer-content-body}}
+          {{/drawer-content}}
+
+          <!-- Panel -->
+          {{#> drawer-panel
+            drawer-panel--type="div"
+            drawer-panel--attribute=(concat 'id="' master-detail-template--id '-panel" aria-label="Panel"')
+            drawer-panel--modifier="pf-m-width-75-on-2xl"
+            drawer-panel--NoBody="true"
+            }}
+
+            <!-- Panel  -->
+            {{#> drawer-panel-body}}
+              {{> master-detail-content-header
+                master-detail-content-header--title="Patternfly-elements"
+                master-detail-content-header--sub-title="PatternFly elements"
+                }}
+            {{/drawer-panel-body}}
+            {{#> drawer-panel-body}}
+              {{> master-detail-content-body}}
+            {{/drawer-panel-body}}
+          {{/drawer-panel}}
+        {{/drawer-main}}
+      {{/drawer}}
+    {{/card}}
+  {{/page-main-section}}
+{{/master-detail-template}}
+```
+
+To initiate independently scrollable content and panel sections simply omit `.pf-c-page__section` or other height managed parent element.
```

Comment:
This is neat. I wonder if this should be an enhancement to the page section though - that might be useful in other contexts, too. like `pf-m-scrollable` that sets `overflow: auto` on the page section. You could define a header or some content in the first page section, then the next page section scrolls, versus the whole content area scrolling. wdyt?